src/ingestion/models/ingestionManager.ts

```diff
@@ -38,6 +39,7 @@ export class IngestionManager {
       fileName: __filename,
       class: IngestionManager.name,
     };
+    this.ingestionJobTypes = this.config.get<string[]>('forbiddenTypesForParallelIngestion');
```

Comment:
rename ingestionJobTypes to forbiddenJobTypes

Reply:
done